e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -270,4 +270,28 @@ describe("viem send tx", () => {
 			}
 		}
 	}).timeout(10_000);
+
+	it("send fee currency tx with just high enough gas price", async () => {
+		const block = await publicClient.getBlock({});
+		// Actually, the base fee will be a bit lower next block, since our blocks
+		// are always mostly empty. But the difference will be much less than the
+		// exchange rate of 2, so the test will still check that we take the fee
+		// currency into account everywhere.
+		const maxFeePerGas = block.baseFeePerGas / 2n;
+		const request = await walletClient.prepareTransactionRequest({
```

Comment:
this is just for this test right? ie i shouldnt be dividing base Fee by 2 normally

Reply:
Yeah, it's a little unclear from the code but `block.baseFeePerGas` is denominated in `CELO` and the [exchange rate](https://github.com/celo-org/op-geth/pull/244/files/3cc1577c27a55eae86a33792b5e4fbd208ffcc3f#diff-186a188150620d50ea72db7fc5bd1d5a9ab3a1dca2a7955ad6912eaba58245a4R45) of the `FEE_CURRENCY2` is `0.5`, so in order to calculate the `maxFeePerGas` high enough to be accepted we divide the base fee by 2.